src/components/Envelope.vue

```diff
@@ -124,18 +124,17 @@
 					@click.prevent="onToggleJunk">
 					<template #icon>
 						<AlertOctagonIcon
-							:title="data.flags.$junk ? t('mail', 'Mark not spam') : t('mail', 'Mark as spam')"
 							:size="20" />
 					</template>
 					{{
 						data.flags.$junk ? t('mail', 'Mark not spam') : t('mail', 'Mark as spam')
 					}}
 				</ActionButton>
-				<ActionButton :close-after-click="true"
+				<ActionButton v-if="hasSelectAcl"
+					:close-after-click="true"
```

Comment:
This can be reverted. The select ACL is corresponding to something different (in the backend only).

It can be ignored in the frontend.